test/tests_for_integration/api_to_db_test.py

```diff
@@ -39,6 +43,13 @@
 # may need to make this configurable
 JARS_DIR = Path("/opt/jars/lib/jars")
 
+USER_READ_ADMIN = "readuser"
+TOKEN_READ_ADMIN = None
```

Comment:
In some programming cultures, variables in all caps are considered to be constants. Does ee2 use any such programming practices? It may be confusing to have them overwritten a little later in the file.

Reply:
I see where you're coming from, but in the context of the tests they are constants. In the context of the fixtures, of course, they aren't. To me it makes a bit more sense to have the user and token both in all caps since they don't change during the tests but there's some grey here.

Reply:
Yeah, that's pretty much the standard here. Constants are basically by convention in Python, like the all caps here, and what's imported from `EE2Constants`